website/docs/r/extension.html.markdown

```diff
@@ -50,6 +50,19 @@ resource "pagerduty_extension" "slack"{
   endpoint_url = "https://generic_webhook_url/XXXXXX/BBBBBB"
   extension_schema = "${data.pagerduty_extension_schema.webhook.id}"
   extension_objects    = ["${pagerduty_service.example.id}"]
+
+  config = <<EOF
+{
+	"restrict": "any",
+	"notify_types": {
+			"resolve": false,
+			"acknowledge": false,
+			"assignments": false
+	},
+  "access_token": "XXX"
```

Comment:
No biggie but the indentation here looks a bit off :)

Reply:
Fixed.